www/js/diary/list/DateSelect.tsx

```diff
@@ -71,20 +71,20 @@ const DateSelect = ({ tsRange, loadSpecificWeekFn }) => {
       <Text>{dateRangeEnd}</Text>
     </NavBarButton>
     <DatePickerModal locale={i18next.resolvedLanguage || 'en'}
-      accessibilityLabel="Select a date"
```

Comment:
Note to self, this is also handled in the cleanup commit
eedb92ce96580238ca4444328560275ed4f84b1

due to
> --DatePickerModal doesn't actually support the accessibilityLabel prop. However, looking through the source code revealed that RN Paper Dates does give much consideration to accessibility. So I think this is handled sufficiently, internally in the library.